hilgendorf_moveit_config/config/joint_limits.yaml

```diff
@@ -0,0 +1,61 @@
+joint_limits:
+  right_wrist_2_joint:
+    has_velocity_limits: true
+    max_velocity: 3.2
+    has_acceleration_limits: false
+    max_acceleration: 0
```

Comment:
I've found a value of: `max_acceleration: 5` is about the highest that the driver can support.  If we use this value, then we should add a note to indicate that the true max acceleration is much higher.